apps/back/src/Service/BalanceService.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Group;
+use Doctrine\ORM\EntityManagerInterface;
+
+class BalanceService
+{
+    public function __construct(public EntityManagerInterface $em)
```

Comment:
1. This parameter should not be public.
2. You can use this instructions to set it
```suggestion
    public function __construct(private readonly EntityManagerInterface $em) 
    {
    }
```